docs/publications/BioHackrXiv/paper.md

```diff
@@ -0,0 +1,215 @@
+
+Crucially, the pipeline can filter its inputs to only include tools that are relevant to a specific research community. 
+Based on the selected filters, a community-specific interactive table is generated that can be embedded, e.g. into the respective [Galaxy Hub](https://galaxyproject.org/) webpage or [Galaxy subdomain](https://galaxyproject.org/eu/subdomains/). 
+This table allows further filtering and searching for fine-grained tool selection. 
+The pipeline is fully automated and executes on a weekly basis. 
+Any scientific community can apply the pipeline to create a table specific to their needs.
+
+An interactive table that presents metadata is only as useful as the metadata annotations it is capturing. 
+To improve the metadata coverage for the interactive table, the project also directly addressed the quality of tool annotations in bio.tools for the microGalaxy community: 
+a community with a focus on tools related to microbial research. 
+
+Annotation guidelines were established for this purpose, the process of updating Galaxy tool wrappers to include bio.tools identifiers was started and the outcome of these activities was evaluated using a crowdsourced approach. During the BioHackathon Europe 2023 week, the annotation practices were applied to the tools selected from the microGalaxy community. 
+This effort allowed the team to connect more than 50 tools to their respective bio.tools entry, update the registry entry, and collectively peer-review the results. 
+
+The established pipeline and the annotation guidelines can support any scientific community to make their Galaxy tools more findable, visible, comparable, understandable, and accessible. 
+Here, we describe the methods and processes that resulted from this project, and highlight how this will now allow the microGalaxy community to confidently navigate an ever-expanding landscape of research software in the Galaxy framework.
+
+
+# Methods
+
+## Domain-specific interactive tools table
+
+To create the domain-specific interactive tools table, Galaxy tool-wrapper suites are first parsed from across multiple GitHub repositories. 
+In effect, the repositories monitored by the planemo-monitor [@citesAsAuthority:Bray2022.03.13.483965] are scraped using a custom script. 
+The planemo-monitor is part of the Galaxy tool-update infrastructure, and keeps track of the most up-to-date tool development repositories.
+
+Metadata is extracted from each parsed tool-wrapper suite. 
+This includes: wrapper suite ID, scientific category, Bioconda dependency, and a repository URL from bio.tools. 
+As a tool suite can be composed of multiple individual tools, the tool IDs for each tool are also extracted.
+The bio.tools reference is used to request metadata annotations via the bio.tools API, including bio.tools description and functionality annotation (i.e. Operation(s)) using EDAM ontology concepts[@usesDataFrom:black2021edam]. 
```

Comment:
```suggestion
The bio.tools reference is used to request metadata annotations via the bio.tools API, including bio.tools description and functionality annotation using EDAM ontology concepts[@usesDataFrom:black2021edam]. 
```
I think this would be more confusing than explanatory.

If you think we need to go more into details of what kind of EDAM annotations, then there should IMHO be a separate sentence about them.